packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -84,21 +84,26 @@ export class VaAccordionItem {
   }
 
   render() {
+    // IE11 is unable to run onSlotChange
+    // Check needed to populate Accordion Item Header information
+    // When using slot="headline" to set the information
+    const ieSlotCheckTag = this.el.querySelector('[slot="headline"]')?.tagName.toLowerCase()
+    const ieSlotCheckHeader = this.el.querySelector('[slot="headline"]')?.innerHTML
```

Comment:
Is it possible to add tests for this additional logic or are we going to run into the the difficulties we've had before with writing tests for IE11 stuff?

Reply:
@k80bowman so the e2e testing suite that Stencil uses is powered by Puppeteer which runs their tests via headless Chrome so trying to run a specific suite through IE11 I don't believe is possible.

Overall everything being done is to bulletproof the code further. Anything that fails via the optional chaining it will return `undefined` and thus fall back to the next value in the OR statement which in this case would be the prop of `this.level` and the prop of `this.header` which is shown on lines 95 and 103

Reply:
That's fine, I knew that we might have problems with Puppeteer. I just wanted to check if adding tests would be possible. Especially with bug fixes I like to make sure we have tests to prevent any future regressions.